[Little Things] URL Bar Impression

bistaastha opened the pull request:
Following changes are associated with this pull request:
- Addition of links to [apis/impressions/url-bar.mdx](https://github.com/firecampapp/firecamp-doc/blob/master/source/apis/impressions/url-bar.mdx)
- Creation of assets folder on the same level as actions as impressions
- Replacing a placeholder with a URL bar screenshot from API app

vipulgupta2048 commented on source/apis/impressions/url-bar.mdx:
```diff
@@ -29,27 +30,27 @@ Request methods are used by protocols like `HTTP` at the times of request to ind
 
-- API Endpoints
-- Request Methods
-- Environment Snippets
-- Projects in Firecamp
```
I would say to not remove them at least because they are going to be deployed sometime. Best to keep them, we can link them soon.

vipulgupta2048 commented on source/apis/impressions/url-bar.mdx:
```diff
@@ -29,27 +30,27 @@ Request methods are used by protocols like `HTTP` at the times of request to ind
 - **UNLINK** - This `UNLINK` method deletes metainformation about an object. Only headers exactly matching the headers given are removed.
 
-You can make a selection of your desired Request Method from the `Quick Method Selection` section availabe in the body tab of Request Panel.
+You can make a selection of your desired Request Method from the `Quick Method Selection` section availabe in the body tab of [**Request Panel**](./request-panel).
```
Do these links work without the .mdx format?

bistaastha replied:
I'm not sure about that. I guess this will be cleared up after deployment?

vipulgupta2048 replied:
Yeah, now that we know about linking in .mdx, can we replace these. Thanks!

vipulgupta2048 commented on source/apis/impressions/url-bar.mdx:
```diff
@@ -29,27 +30,27 @@ Request methods are used by protocols like `HTTP` at the times of request to ind
+<!--TODO: Add links to Environment Snippets &  Projects in Firecamp-->
 ## Save Button
-The `Save` button in the URL bar saves your request to the project/module for future use by you or your team. Learn more about **Projects in Firecamp** and **Steps to save your request in Firecamp**.
+The `Save` button in the URL bar saves your request to the project/module for future use by you or your team. Learn more about **Projects in Firecamp** and [**Steps to save your request in Firecamp**](../actions/saving-your-requests).
```
Not the right link, for saving any request for that matter here's the journalized one that I have created. Which is specific about saving https://github.com/firecampapp/firecamp-doc/blob/master/source/discover%20firecamp/project%20management/saving-your-first-request.mdx

bistaastha replied:
I'll add this one to TODO then.

vipulgupta2048 replied:
Cool, get it done.

vipulgupta2048 commented on source/apis/impressions/url-bar.mdx:
```diff
@@ -6,20 +6,21 @@ description: Overview of the URL Bar in the API client
+The URL bar consists of 5 elements:
 
-The URL bar consists of 5 elements,
 1. Method selection
 2. URL field
 3. Send button (Execute)
 4. Save button
 5. Environment badges
```
Need Anchor tags for these.

vipulgupta2048 commented on source/apis/impressions/url-bar.mdx:
```diff
@@ -6,20 +6,21 @@ description: Overview of the URL Bar in the API client
+Request Methods are used by protocols like `HTTP` at the times of request to indicate the desired action that needs to be performed for a given resource. Firecamp supports several request methods that act on requests made to the API endpoint. The following methods are available for use through the dropdown in the URL bar:
 
 - **GET** - The `GET` method is used to retrieve data and should have no other effect on the data.
-- **POST** - ****The `POST` method is used to submit an entity to the specified resource, often causing a change in state or modify the data.
```
Do remove this change. 
@Sc0rpi0n101 has already made this change in #31

Sc0rpi0n101 commented:
@vipulgupta2048 Which ones of these do you think looks better for the URL bar? 

![image](https://user-images.githubusercontent.com/30147327/78472393-e79d6980-7755-11ea-92ca-e2a9d33e8662.png)


![image](https://user-images.githubusercontent.com/30147327/78472509-018b7c00-7757-11ea-84a0-3f9c211c8438.png)

How's this for the method selection?

![image](https://user-images.githubusercontent.com/30147327/78472450-7c07cc00-7756-11ea-9fca-f450669623ee.png)

Also, we have `blue` send button, not an `orange` one as told in the docs?

vipulgupta2048 commented:
@Sc0rpi0n101 Thanks for bringing it my notice, probably when we were writing it. It was orange. Replace it with the right color.

From the 2 screenshots you sent regarding the URL bar, I feel the first one could be a good one to have. @Nishchit14 Can you also tell us about your preference. 

Regarding, Method selection ss, I would say go easy on the blur. Otherwise looks good.

bistaastha commented:
Some things I did on this commit:
- Added numbers to each heading and removed the list in the beginning, since they can be navigated through the side navigation panel anyway.
- Added a more compressed version of the image :)
- Used the `Link` component. I was not able to test this out on my own system because of some other things which are not working out fine in my environment yet.

Nishchit14 commented:
@Sc0rpi0n101 @vipulgupta2048 

The first image is perfect for spotting URLbar.

vipulgupta2048 commented:
@bistaastha @Sc0rpi0n101 Make the required changes + start applying the little things list to the docs, starting with API.

vipulgupta2048 commented:
@bistaastha @Sc0rpi0n101 Make the required changes + start applying the little things list to the docs, starting with API.

Nishchit14 commented:
Thank you @bistaastha and @Sc0rpi0n101 for this PR...  I am merging it.
